pokedex.css

```diff
@@ -0,0 +1,48 @@
+html {
+  width: 100vw;
+  height: 100vh;
+  box-sizing: border-box;
+}
+.bottom-half {
+  background-color: #c4c4c4;
+  width: 100%;
+  height: 47.5%;
+  background-position: bottom;
+}
+.middle-bar {
+  background-color: #695b5b;
+  width: 100%;
+  height: 5%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+}
+.middle-circle {
+  background-color: #695b5b;
+  border-radius: 100%;
+  height: 325%;
+  width: 25%;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+}
+.inner-circle {
+  background-color: #c4c4c4;
+  border-radius: 100%;
+  height: 80%;
+  width: 80%;
+  display: flex;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border: solid black 2px;
+}
+.top-half {
+  background-color: #cb2424;
+  width: 100%;
+  height: 47.5%;
+  background-position: top;
+}
+@media screen and (min-width: 992px) {
```

Comment:
You should not include future state in your current ticket.  This should be removed if it's not being used right now.